Sources/Features/AssetsFeature/AssetsView+Reducer.swift

```diff
@@ -31,6 +32,7 @@ public struct AssetsView: Sendable, FeatureReducer {
 
 		public let account: Profile.Network.Account
 		public var isLoadingResources: Bool = false
+		public var isRefreshing: Bool = false
```

Comment:
Do we use `isRefreshing` in the View? I dont think so? Should be?

Reply:
It is used to tell the pool units child to refresh itself. I am not that happy with this, we should probably have separate lists per tab, instead of one in AssetsView, allowing them to be refreshed separately.